docs/LINKS.md

```diff
@@ -34,3 +34,6 @@
 
 - [rtl_snr](https://github.com/hdtodd/rtl_snr): 
   snr is a pair of equivalent C and Python programs that catalog and analyze signal-to-noise ratios from devices seen by RTL_SDR dongles and logged in JSON format by rtl_433.
+
+- [rtl_sdr driver](https://f-droid.org/en/packages/marto.rtl_tcp_andro/):
+  rtl_sdr driver is a port of rtl_tcp to Android.  See also the [source code](https://github.com/signalwareltd/rtl_tcp_andro-).
```

Comment:
This appears to be an unintended change.  It might be good to rebase into one squashed commit or a series of clean changesets, to make review easier.    Note that it' is my personal opinion that PRs should be rebaesd into clean changesets, even though it is not project doctrine.   (Project doctrine, I try to remind myself, is to squash on merge.)

Reply:
Yes, not sure what happened there, I did not make that change. I agree a rebase would be better but in this case some change was merged in instead. I already made sure to put those changes that could conflict (registration in cmake and rtl_433_devices.h) as a separate commit so it would be easy to rebase. I can do the rebase push again, but perhaps @zuckschwerdt intended something else here.